charts/all/rhods-operator/templates/rhods-datasciencecluster.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: datasciencecluster.opendatahub.io/v1
```

Comment:
Similar comment about potentially parameterizing - parts of this seems like they could usefully be parameters to enhance re-usability